src/test/java/teammates/test/cases/action/InstructorFeedbackCopyActionTest.java

```diff
@@ -89,10 +89,10 @@ public void testExecuteAndPostProcess() throws Exception {
                 + "instr1@course1.tmt|||New Feedback Session "
                 + "<span class=\"bold\">(Copied Session)</span> for Course "
                 + "<span class=\"bold\">[idOfTypicalCourse1]</span> created.<br>"
-                + "<span class=\"bold\">From:</span> Sun Apr 01 23:59:00 UTC 2012"
-                + "<span class=\"bold\"> to</span> Fri Apr 30 23:59:00 UTC 2027<br>"
-                + "<span class=\"bold\">Session visible from:</span> Wed Mar 28 23:59:00 UTC 2012<br>"
-                + "<span class=\"bold\">Results visible from:</span> Sat May 01 23:59:00 UTC 2027<br><br>"
+                + "<span class=\"bold\">From:</span> Sun Apr 01 21:59:00 UTC 2012"
```

Comment:
This one shifts back by only 2 hours. I suppose this is because the test case is configured as UTC +2?

Reply:
Yup it's specified here:
https://github.com/TEAMMATES/teammates/blob/c18c8b3e762558e590d6997e3bde098968ae7cdc/src/test/resources/data/typicalDataBundle.json#L617-L639